packages/core/src/styling/variantSelector.ts

```diff
@@ -98,6 +101,7 @@ export type ElementNodeModel = {
   attrs: Record<string, Formula>
   style: NodeStyleModel
   variants?: StyleVariant[]
+  animations?: Record<string, Record<string, AnimationKeyframe>>
```

Comment:
The `ElementNodeModel` and `ComponentNodeModel` types in this file seem almost duplicate of the ones in `packages/core/src/component/component.types.ts`? 🤔